--- cmake/platforms/Emscripten-clang/config.cmake	2020-11-23 13:44:00.212457094 +0100
+++ cmake/platforms/Emscripten-clang/config.cmake.new	2020-11-23 13:41:37.740930118 +0100
@@ -1,5 +1,5 @@
 set(VORPALINE_ARCH_64 true)
 include(${GEOGRAM_SOURCE_DIR}/cmake/platforms/Emscripten-clang.cmake)
-add_flags(CMAKE_CXX_FLAGS -m64)
-add_flags(CMAKE_C_FLAGS -m64)
+# add_flags(CMAKE_CXX_FLAGS -m64)
+# add_flags(CMAKE_C_FLAGS -m64)
 
